crates/proof-of-sql/tests/integration_tests.rs

```diff
@@ -160,6 +160,81 @@ fn we_can_prove_a_basic_inequality_query_with_curve25519() {
     assert_eq!(owned_table_result, expected_result);
 }
 
+#[test]
+#[cfg(feature = "blitzar")]
+fn we_can_prove_a_basic_query_containing_extrema_with_curve25519() {
+    let mut accessor = OwnedTableTestAccessor::<InnerProductProof>::new_empty_with_setup(());
+    accessor.add_table(
+        "sxt.table".parse().unwrap(),
+        owned_table([
+            smallint("smallint", [i16::MIN, i16::MAX]),
+            int("int", [i32::MIN, i32::MAX]),
+            bigint("bigint", [i64::MIN, i64::MAX]),
```

Comment:
can we please include zero in these tests, Im also not sure if its worth testing the other numerical boundaries as well such as i128

Reply:
decimal75 boundaries would be whatever the scalar min max are I suppose

Reply:
```suggestion
            smallint("smallint", [i16::MIN, 0, i16::MAX]),
            int("int", [i32::MIN, 0, i32::MAX]),
            bigint("bigint", [i64::MIN, 0, i64::MAX]),
```